html/api_v0.php

```diff
@@ -186,6 +186,7 @@ function () use ($app) {
                     '/services',
                     function () use ($app) {
                         $app->get('/:hostname', 'authToken', 'list_services')->name('get_service_for_host');
+                        $app->post('/:hostname', 'authToken', 'add_service_for_host')->name('add_service_for_host');
```

Comment:
I'd change `add_service_for_host` to just `add_service`.

Reply:
It's already defined in https://github.com/librenms/librenms/blob/master/includes/services.inc.php#L37 and used in the function

Reply:
Good shout :)